circle.yml

```diff
@@ -13,6 +13,7 @@ dependencies:
     - sudo apt-get install python-numpy python-scipy python-dev python-matplotlib
     - sudo apt-get install python-nose python-coverage
     - sudo apt-get install python-sphinx
+    - pip install --find-links http://wheels.astropy.org/ --find-links http://wheels2.astropy.org/ --use-wheel Cython
```

Comment:
That's not obvious, that's because they are the only ones available (they
do not exist in Pypi) - pandas use them as well. I should add a comment
there
Le 3 nov. 2015 18:32, "Manoj Kumar" <notifications@github.com> a écrit :

> In circle.yml
> <https://github.com/scikit-learn/scikit-learn/pull/5492#discussion_r43779221>
> :
>
> > @@ -13,6 +13,7 @@ dependencies:
> >      - sudo apt-get install python-numpy python-scipy python-dev python-matplotlib
> >      - sudo apt-get install python-nose python-coverage
> >      - sudo apt-get install python-sphinx
> > +    - pip install --find-links http://wheels.astropy.org/ --find-links http://wheels2.astropy.org/ --use-wheel Cython
>
> Sorry if this is obvious, but why are we depending on astropy's wheels to
> install Cython?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/5492/files#r43779221>.
>